### DISCUSSION TITLE
showGridlines not working

### DISCUSSION THREAD

**Discussion by msagle** [primefaces/community#366] [PrimeReact] [answered]
Hi Prime team!

I'm working with the PrimeReact DataTable for a React-based project written in TS, and I cannot get any gridlines to show up on my DataTable; in essence, "showGridlines" is not having any effect. I've combed through many stack overflow postings and can't seem to find a solution. Any help would be appreciated.

Here are a few of my dependencies  (listing the ones that I feel like *could* be invalidating showGridlines)-
"dependencies" : {
        ......
        "bootstrap": "^5.3.2",
        "primeicons": "^6.0.1",
        "primereact": "^10.0.9",
        "react": "18.2.0",
        "react-bootstrap": "^2.9.1",
        "reactstrap": "^9.1.8"
       ......
}

Also, I'm relatively new to frontend work, so anything that you feel might be obvious as solution might not be obvious to me!

**Comment by melloware** [comment, marked as answer]
Works fine see this reproducer and compare with what you are doing: https://stackblitz.com/edit/react-uvi2x9?file=src%2FApp.js

**Reply by msagle** [reply]
Thank you for your response... unfortunately, I've compared and everything looks the same to me. Even my dependencies have the same version. I do have bootstrap/react-bootstrap in the package.json file, but neither are currently being utilized in the project... is that a potential issue?

**Reply by melloware** [reply]
Has to be something causing a conflicting CSS style so I would pull that out and try it.

**Reply by msagle** [reply]
yep, that was it!!! Many thanks!!!!!
